2019-08-28 15:16:58,726 - [INFO] - [msg]开始执行用例 - test_01_register.py - 30
2019-08-28 15:17:02,297 - [INFO] - [msg]用例执行结束 - test_01_register.py - 36
2019-08-28 15:17:02,297 - [INFO] - [msg]开始执行用例 - test_02_login.py - 29
2019-08-28 15:17:03,345 - [INFO] - [msg]用例执行结束 - test_02_login.py - 34
2019-08-28 15:17:03,464 - [INFO] - [msg]开始执行用例 - test_03_recharge.py - 33
2019-08-28 15:17:04,248 - [INFO] - [msg]用例执行结束 - test_03_recharge.py - 40
2019-08-28 15:17:04,358 - [INFO] - [msg]开始执行用例 - test_04_add.py - 32
2019-08-28 15:17:04,757 - [INFO] - [msg]用例执行结束 - test_04_add.py - 39
2019-08-28 15:17:04,866 - [INFO] - [msg]开始执行用例 - test_05_invest.py - 32
2019-08-28 15:17:06,348 - [ERROR] - [msg]测试投资人投资，loanId为负数, 执行结果为：Fail具体异常为：'11005' not found in '{"status":0,"code":"11009","data":null,"msg":"不存在该标的"}' : 测试投资人投资，loanId为负数 - test_05_invest.py - 83
2019-08-28 15:17:06,868 - [INFO] - [msg]用例执行结束 - test_05_invest.py - 39
